eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/BackPropagatedRSWriter.java

```diff
@@ -205,31 +148,15 @@ else if (ssInRemappedAssemblyGroupedByID.containsKey(ssIDToBeClustered)) {
                     numDbsnpRSAssignments += 1;
                 }
 
-                if (Objects.nonNull(newRSRecordForOriginalAssembly)) {
-                    if (clusteringWriter.getClusteredVariantCollection(rsToAssign).equals(
-                            ClusteredVariantEntity.class)) {
-                        bulkCVEInserts = evaCVEInserts;
-                        numEVARSInserts += 1;
-                    } else {
-                        bulkCVEInserts = dbsnpCVEInserts;
-                        numDbsnpRSInserts += 1;
-                    }
-                    bulkCVEInserts.insert(newRSRecordForOriginalAssembly);
-                    metricCompute.addCount(ClusteringMetric.CLUSTERED_VARIANTS_CREATED, 1);
-                }
-
                 Query queryToLookUpSSHash = query(where(ID_ATTRIBUTE).is(submittedVariantEntity.getHashedMessage()));
-                Update updateRSInOriginalAssembly = Update.update(RS_ATTRIBUTE, rsToAssign);
+                Update updateRSInOriginalAssembly = Update.update(BACKPROP_RS_ATTRIBUTE, rsToBackPropagate);
                 bulkSVEUpdates.updateOne(queryToLookUpSSHash, updateRSInOriginalAssembly);
-                metricCompute.addCount(ClusteringMetric.SUBMITTED_VARIANTS_CLUSTERED, 1);
 
-                bulkSVOEInserts.insert(getUpdateOperation(submittedVariantEntity, rsToAssign));
+                bulkSVOEInserts.insert(getUpdateOperation(submittedVariantEntity, rsToBackPropagate));
```

Comment:
I don't think this is the same type of operation as a clustering. I think it might end up being misleading to associated back propagation and clustering considering that we decided to keep them separated.

Reply:
The UPDATED eventType is a generic event that can signify many events (see [here](https://github.com/EBIvariation/eva-accession/blob/e1502acb32ddb4ad41807e94052679cdf8b85241/eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/RSSplitWriter.java#L272) and [here](https://github.com/EBIvariation/eva-accession/blob/e1502acb32ddb4ad41807e94052679cdf8b85241/eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/RSMergeWriter.java#L469) for example). Usually the reason attribute is the one that carries event-specific information.

Reply:
I do kind of like having more specific event types (as we do with the clustered variant operations) rather than a generic event, but probably a discussion for another day...

Reply:
My point is that we're not updating the `rs` field here but the `backPropagatedRS` field which is what all other UPDATED operation are. 
April's point also stands that having the origin of the update would avoid situation like [this one](https://github.com/EBIvariation/eva-accession/blob/7b5f55371a6bc50361d5008ded500a2f77751947/eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/merged/MergedVariantMongoReader.java#L160) where we need to check the reason to figure out where it came from.

Reply:
Addressed [here](https://github.com/EBIvariation/accession-commons/pull/74/files)